tools/ovc/openvino/__init__.py

```diff
@@ -75,12 +90,30 @@
 from openvino import opset15
 from openvino import opset16
 
+# Extend Node class to support binary operators
+Node.__add__ = opset13.add
+Node.__sub__ = opset13.subtract
+Node.__mul__ = opset13.multiply
+Node.__div__ = opset13.divide
+Node.__truediv__ = opset13.divide
+Node.__radd__ = lambda left, right: opset13.add(right, left)
+Node.__rsub__ = lambda left, right: opset13.subtract(right, left)
+Node.__rmul__ = lambda left, right: opset13.multiply(right, left)
+Node.__rdiv__ = lambda left, right: opset13.divide(right, left)
+Node.__rtruediv__ = lambda left, right: opset13.divide(right, left)
+Node.__eq__ = utils.deprecated(version="2025.3", message="Use ops.equal instead")(opset13.equal)
+Node.__ne__ = utils.deprecated(version="2025.3", message="Use ops.not_equal instead")(opset13.not_equal)
+Node.__lt__ = utils.deprecated(version="2025.3", message="Use ops.less instead")(opset13.less)
+Node.__le__ = utils.deprecated(version="2025.3", message="Use ops.less_equal instead")(opset13.less_equal)
+Node.__gt__ = utils.deprecated(version="2025.3", message="Use ops.greater instead")(opset13.greater)
+Node.__ge__ = utils.deprecated(version="2025.3", message="Use ops.greater_equal instead")(opset13.greater_equal)
+
```

Comment:
we don't need it here. You can extend it in one place and re-use it overall.
Let say in py file for Node class that is more logical to have

Reply:
I fully agree with your comment. I created [a task](https://jira.devtools.intel.com/browse/CVS-159588) to improve this.
This PR is simply about moving functionality from` openvino/runtime/__init__`  to `openvino/__init__` so let's discuss it as a separate issue.

Reply:
Let us do in this PR. It is not a big deal